ui/pages/onboarding-flow/secure-your-wallet/skip-srp-backup-popover.js

```diff
@@ -59,15 +59,13 @@ export default function SkipSRPBackup({ handleClose }) {
               handleClose();
             }}
             type="secondary"
-            rounded
           >
             {t('goBack')}
           </Button>
           <Button
             data-testid="skip-srp-backup"
             disabled={!checked}
             type="primary"
-            rounded
```

Comment:
Removing rounded prop from Buttons that use it. Also did a search for remaining instances and didn't see any

<img width="462" alt="Screenshot 2025-05-17 at 10 14 53 AM" src="https://github.com/user-attachments/assets/72d4cf3a-ff31-431a-af29-296d4e99728d" />